File: ApiSuvesaPos/Negocio/Modelo/cxp/abonocpagar/abonocpagar.vb
Namespace Modelo
    Public Class abonocpagar
        Public Property Id_Abonocpagar As Long
        Public Property Documento As Long
        Public Property TipoDocumento As String
        Public Property CuentaBancaria As String
        Public Property Codigo_banco As Long
        Public Property Saldo_Cuenta As Decimal
        Public Property Monto As Decimal
        Public Property Saldo_Actual As Decimal
        Public Property Fecha As DateTime
        Public Property Contabilizado As Boolean
        Public Property Recibo As String
        Public Property Cedula_Usuario As String
        Public Property Cod_Proveedor As Integer
        Public Property Anulado As Boolean
        Public Property cod_Moneda As Integer
        Public Property TipoCambio As Decimal
        Public Property MontoLetras As String
        Public Property CuentaDestino As Long
        Public Property FechaEntrada As DateTime
        Public Property Observaciones As String

    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/cxp/abonocpagar/abonocpagar.cs
namespace Modelo
{
    public class abonocpagar
    {
        public long Id_Abonocpagar { get; set; }
        public long Documento { get; set; }
        public string TipoDocumento { get; set; }
        public string CuentaBancaria { get; set; }
        public long Codigo_banco { get; set; }
        public decimal Saldo_Cuenta { get; set; }
        public decimal Monto { get; set; }
        public decimal Saldo_Actual { get; set; }
        public DateTime Fecha { get; set; }
        public bool Contabilizado { get; set; }
        public string Recibo { get; set; }
        public string Cedula_Usuario { get; set; }
        public int Cod_Proveedor { get; set; }
        public bool Anulado { get; set; }
        public int cod_Moneda { get; set; }
        public decimal TipoCambio { get; set; }
        public string MontoLetras { get; set; }
        public long CuentaDestino { get; set; }
        public DateTime FechaEntrada { get; set; }
        public string Observaciones { get; set; }
    }
}
